hera_pspec/pspecdata.py

```diff
@@ -2616,11 +2633,15 @@ def pspec_run(dsets, filename, dsets_std=None, groupname=None, dset_labels=None,
     history : str
         String to add to history of each UVPSpec object.
 
-    Returns
+    Returns (psc, ds)
```

Comment:
This will break readthedocs

Reply:
oh okay, so I'll just leave it blank then I guess